packages/api-explorer/style/main.scss

```diff
@@ -0,0 +1,990 @@
+  width: 100%;
+}
+
+/* Remove left-margins from rows that are offset with Bootstrap's grid system. */
+form.rjsf .row .col-xs-offset-9 {
+  margin-left: 0px;
+}
+
+/* Override Bootstrap properties for "add" button (for arrays). */
+.array-item-add.col-xs-3 {
+  flex-basis: unset;
+  max-width: unset;
+  margin-top: 10px;
+}
+
+/* For up/down/remove toolbox for items in arrays, change default size. */
+.array-item-toolbox.col-xs-3 {
+  flex-basis: 12%;
+}
+
+/* Align up/down/remove toolbox to the right. */
+.array-item-toolbox .btn-group {
+  justify-content: flex-start !important;
+}
+
+/* Reset padding and remove border + other styles from buttons in up/down/remove toolbox. */
+.array-item-toolbox .btn {
+  padding-left: 8px;
+  padding-right: 8px;
+  border: none;
+  width: 36px !important;
+  height: 36px !important;
+  flex: unset !important;
+  font-size: 13px;
+  background: rgba(255, 255, 255, 0);
+}
+
+form.rjsf .btn:active {
+  box-shadow: none;
+}
+
+form.rjsf .btn:focus {
+  outline: none;
+}
+
+form.rjsf .btn.btn-default {
+  border: none;
+}
+
+form.rjsf .btn.btn-default:hover {
+  border: none;
+}
+
+form.rjsf .btn.btn-add {
+  transform: rotate(45deg);
+  font-weight: bold;
+  font-size: 11px;
+  color: rgb(76, 85, 90);
+  background: rgba(255, 255, 255, 0);
+  margin-bottom: 1em;
+}
+
+form.rjsf .btn.btn-add::after {
+  content: '✕';
+  font-size: 11px;
+  font-weight: bold;
+}
+
+form.rjsf .btn.array-item-move-up::after {
+  content: '↑';
+}
+
+form.rjsf .btn.array-item-move-down::after {
+  content: '↓';
+}
+
+form.rjsf .btn.array-item-remove::after {
+  content: '✕';
+}
+
+/* Fix color of up/down buttons when disabled. */
+form.rjsf button[disabled] {
+  color: #d2d2d2 !important;
+}
+
+/* Add type labels. */
+.field.field-array-of-string legend::after {
+  color: rgba(0, 0, 0, 0.6);
+  content: 'array of strings';
+  background: #f2f2f2;
+  border-radius: 3px;
+  font-family: 'Proxima Nova', sans-serif;
+  font-size: 11px;
+  font-weight: lighter;
+  margin-left: 10px;
+  padding: 3px 6px;
+}
+
+.field.field-array-of-integer legend::after {
+  color: rgba(0, 0, 0, 0.6);
+  content: 'array of integers';
+  background: #f2f2f2;
+  border-radius: 3px;
+  font-family: 'Proxima Nova', sans-serif;
+  font-size: 11px;
+  font-weight: lighter;
+  margin-left: 10px;
+  padding: 3px 6px;
+}
+
+.field.field-array-of-object > legend::after {
+  color: rgba(0, 0, 0, 0.6);
+  content: 'array of objects';
+  background: #f2f2f2;
+  border-radius: 3px;
+  font-family: 'Proxima Nova', sans-serif;
+  font-size: 11px;
+  font-weight: lighter;
+  margin-left: 10px;
+  padding: 3px 6px;
+}
+
+fieldset[id^='root_array of objects_'] > legend::after {
+  content: 'object';
+  color: rgba(0, 0, 0, 0.6);
+  background: #f2f2f2;
+  border-radius: 3px;
+  font-family: 'Proxima Nova', sans-serif;
+  font-size: 11px;
+  font-weight: lighter;
+  margin-left: 10px;
+  padding: 3px 6px;
+}
+
+.field.field-array-of-boolean legend::after {
```

Comment:
After this is merged in, we should come back through this file and clean up all these similar legend definitions with an `@each` declaration now that this is a Sass.